client/src/store/new-request-module.ts

```diff
@@ -1812,12 +1823,10 @@ export class NewRequestModule extends VuexModule {
       }
       this.mutateDisplayedComponent('AnalyzeResults')
     } catch (error) {
-      // eslint-disable-next-line
-      console.log(error)
-      if (error.code === 'ECONNABORTED') {
+      if (error.code === 'ECONNABORTED' || error.message === 'Network Error') {
         this.mutateNameAnalysisTimedOut(true)
-        this.mutateSubmissionTabComponent('EntityNotAutoAnalyzed')
-        this.mutateDisplayedComponent('SubmissionTabs')
+        this.mutateName(this.name)
+        this.mutateDisplayedComponent('SendToExamination')
```

Comment:
So, if the request times out then we display the Send To Examination page?

Reply:
Yes. It will search upto 2 minutes now. If we are not able to get the result within that time it shows the new generic screen @cameron-freshworks has built (SendToExamination) that says Entity cannot be auto-analyzed. From there the user can navigate to send for review by clicking Continue Button